portfolio/src/main/webapp/index.html

```diff
@@ -19,15 +19,19 @@
         <a href="aboutme.html">about me</a>
       </div>
       <h1>Welcome to Jenny's Portfolio!</h1>
+      <!-- <button onclick="displayElement();">check log in status</button> -->
+      <div id=loginStatus>TESTING USER LOG IN</div>
```

Comment:
rename div?

Reply:
I changed this div id to loginStatusMessage & I'm wondering if that makes more sense. The purpose of this div is to give the user a message to log in/log out based on whether they're logged in or not

Reply:
I think that's fine. Add a comment if you feel like you can't explain things properly with a var name